logstash-core/build.gradle

```diff
@@ -49,9 +49,10 @@ task cleanGemjar {
 clean.dependsOn(cleanGemjar)
 jar.finalizedBy(copyGemjar)
 task gemspec_jars {
+    def gemspecJars = file("${projectDir}/gemspec_jars.rb")
+    outputs.files gemspecJars
```

Comment:
Can you help me understand what outputs.files is doing here vs. the prior implementation ?

Reply:
@jakelandis it ensures that we will rerun this task only if the dependencies change or that file was changed. Just makes the build faster + it's a nice way of documenting what actually happens here (imo).